src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  let basePrice = days * PRICE_PER_DAY;
+
+  if (days >= LONG_TERM) {
+    return (basePrice -= LONG_TERM_DISCOUNT);
+  }
+
+  if (days >= SHORT_TERM) {
+    return (basePrice -= SHORT_TERM_DISCOUNT);
```

Comment:
Similar to the previous comment, avoid using an assignment (`-=`) inside the return statement. Return `basePrice - SHORT_TERM_DISCOUNT` instead.